how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to Whitelist the Elastic Amazon Web Services IPs for Telerik Platform
+description: Establishing a secure connection between your machine and AWS that AppBuilder stays behind.
+type: how-to
+page_title: Whitelisting Amazon IPs for Secure AppBuilder Connection
+slug: how-to-whitelist-aws-ips 
+position:
+tags: platform-core, cloud-services
+teampulseid: 
+ticketid:  922445, 1014483
+pitsid:
+aws: Amazon Web Services
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Telerik Platform</td>	
+  </tr>
+  <tr>
+	<td>Feature</td>
+	<td>Security, Connectivity</td>
+  </tr>
+</table>
+
+## Description
+Is there a way to whitelist the range of IP addresses that Telerik Platform uses on the {{ page.aws }}  (AWS) infrastructure? How to do this having in mind that AWS IPs are elastic? I need this in order to establish a secure communication channel.
```

Comment:
Everything looks good to me but this paragraph. I am bothered by the list of questions and the "I" sentence. Perhaps Evgeni Boevski should take a look as well. 

Keep in mind that if this is the style we're after for the description, all other KBs should use similar phrasing.